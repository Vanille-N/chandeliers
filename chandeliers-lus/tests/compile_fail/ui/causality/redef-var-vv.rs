//! Duplicate local variable definition.
//! For rustc this is just a variable shadowing so we need to catch it manually.
chandeliers_lus::decl! {
    node foo() returns ();
    var a : int; a : int;
    let a = 1; tel
}

fn main() {}
