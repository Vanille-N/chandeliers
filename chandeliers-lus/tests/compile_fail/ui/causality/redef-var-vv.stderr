error: Attempt to redefine a, when a prior item already defines it
 --> tests/compile_fail/ui/causality/redef-var-vv.rs:3:18
  |
3 |     var a : int; a : int;
  |                  ^
  |
note: Already defined here
 --> tests/compile_fail/ui/causality/redef-var-vv.rs:3:9
  |
3 |     var a : int; a : int;
  |         ^
